Update Beam SDK dependency page and linked material. (#30712)
apache/beam: website/www/site/content/en/documentation/sdks/python-dependencies.md

# Beam SDK for Python dependencies

> This page provides the information on dependencies of Apache Beam Python SDK

Reviewer: ```suggestion
This page provides the information about the Apache Beam Python SDK dependencies.
```

> itself.

Reviewer: ```suggestion
```

> If your pipeline requires addtional dependencies, see the [Managing Python Pipeline Dependencies](https://beam.apache.org/documentation/sdks/python-pipeline-dependencies/) page.

Reviewer: ```suggestion
If your pipeline requires additional dependencies, see [Managing Python Pipeline Dependencies](https://beam.apache.org/documentation/sdks/python-pipeline-dependencies/).
```

> Dependencies of the Apache Beam Python SDK are defined in the `setup.py` file in the Beam repository. To view them, perform the following steps:

Reviewer: ```suggestion
Dependencies of the Apache Beam Python SDK are defined in the `setup.py` file in the Beam repository. To view them, take the following steps:
```

1. Open `setup.py`.

    ```
    https://github.com/apache/beam/blob/release-<VERSION_NUMBER>/sdks/python/setup.py
    ```

>     <p class="paragraph-wrap">Replace `&lt;VERSION_NUMBER&gt;` with the major.minor.patch version of the SDK. For example, <a href="https://github.com/apache/beam/blob/release-{{< param release_latest >}}/sdks/python/setup.py" target="_blank" rel="noopener noreferrer">https://github.com/apache/beam/blob/release-{{< param release_latest >}}/sdks/python/setup.py</a> will provide the dependencies for the {{< param release_latest >}} release.</p>

Reviewer: ```suggestion
    <p class="paragraph-wrap">Replace `&lt;VERSION_NUMBER&gt;` with the major.minor.patch version of the SDK. For example, <a href="https://github.com/apache/beam/blob/release-{{< param release_latest >}}/sdks/python/setup.py" target="_blank" rel="noopener noreferrer">https://github.com/apache/beam/blob/release-{{< param release_latest >}}/sdks/python/setup.py</a> provides the dependencies for the {{< param release_latest >}} release.</p>
```

2. Review the core dependency list under `REQUIRED_PACKAGES`.

>     **Note:** If you require [extra features](/get-started/quickstart-py#extra-requirements) such as `gcp` or `dataframe`, you should review the lists under `extras_require` for additional dependencies.

Reviewer: ```suggestion
    **Note:** If you need [extra features](/get-started/quickstart-py#extra-requirements), such as `gcp` or `dataframe`, review the lists in `extras_require` for additional dependencies.
```
